tutoraspects/templates/aspects/apps/superset/pythonpath/locale.yaml

```diff
@@ -1,473 +1,2319 @@
+    in Studio. Once you re-publish your course the new names should appear with no
+    data lost.
+
+
+    ---
+
+
+    Q. Why do I see "No results were returned for this query"?
+
+
+    A. Sometimes the feature we are trying to report on is not enabled for the component,
+    such as a video with no captions or transcripts. In other cases you may need to
+    select a course, video, or problem from the filters before data will populate.
+
+
+    ---
+
+
+    Q. How long does it take data to appear in charts?
+
+
+    A. In most cases the data is updated in under a minute, but you may need to select
+    "Force Refresh" from the "three dot" menu in the top right of a chart to see the
+    newest data. In the future some data may take longer to process, in which case
+    it should be noted in the chart''s description (also accessible from the "three
+    dot" menu).'
+  : ''
+  ? '### Course Enrollment
+
+
+
+    These charts can help you understand how many students are enrolled in your course
+    by showing how enrollments have changed over time and which enrollment modes students
+    are choosing. More details about each chart can be found by clicking the "3 dot"
+    menu at the top right of the chart and selecting "Show chart description".
+
+
+    You can filter these results by choosing selecting various options from the filters
+    on the left.'
+  : ''
+  ? '### Problem Engagement
+
+
+    These charts can help you learn which problems your students have engaged with
+    or skipped, and the relative difficulty of problems across your course. More details
+    about each chart can be found by clicking the "3 dot" menu at the top right of
+    the chart and selecting "Show chart description".
+
+
+    You can filter these results by choosing selecting various options from the filters
+    on the left.'
+  : ''
+  ? '### Problem Performance
+
+
+    These charts help show the difficulty of your problems by showing you which answers
+    your students choose and which require many tries or hints to complete successfully.
+
+
+    Select a problem from the filters on the left to populate the charts.'
+  : ''
+  ? '### Video Engagement
+
+
+    These charts can help understand which videos are watched, and rewatched, most
+    often. It can also show how many users use the closed captions and transcript
+    features on your videos, and how often.
+
+
+    You can filter these results by choosing selecting various options from the filters
+    on the left.'
+  : ''
+  ? '### Video Performance
+
+
+    This section allows you to see which parts of a particular video are most watched
+    or skipped.
+
+
+    Select a problem from the filters on the left to populate the charts.'
+  : ''
+  ? A count of the number of enrollments and un-enrollments per day. Learners can
+    enroll and unenroll multiple times, in this chart each individual enrollment and
+    unenrollment will be counted.
+  : ''
```

Comment:
Does Transifex support the text of this size?

Do you have any ideas to break this down?

Reply:
We can make each question its own block of text. I don't know of limits on Transifex's side aside from ones set by devs on individual strings, but translators would probably appreciate it. I'll try it.